CMakeLists.txt

```diff
@@ -1,18 +1,23 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(eigen_stl_containers)
 
-find_package(catkin REQUIRED)
+if(NOT WIN32)
```

Comment:
This block is redundant (and actually somewhat in opposition to):

```
if(NOT CMAKE_CXX_STANDARD)
  set(CMAKE_CXX_STANDARD 14)
endif()
```

below, so just remove it.